docs/best-practice-non-memory-safe-by-default-languages.md

```diff
@@ -0,0 +1,12 @@
+# Best Practices - Non Memory-Safe By Default Languages
+
+This working group recommends switching to a memory-safe by default language whenever possible or practical. However, when switching to a memory-safe by default language is not possible or practical, there are developer practices that will improve memory safety, even if it is not the default.
+

```

Comment:
should this also point to the BEST c/c++ hardening guide: https://github.com/ossf/wg-best-practices-os-developers/tree/main/docs/Compiler-Hardening-Guides

Reply:
Done!